map.js

```diff
@@ -1,17 +1,18 @@
 (function() {
 	'use strict';
-	let mymap;
 	kintone.events.on('app.record.detail.show', function(event) {
 		const record = event.record;
 		const address = record.Address.value;
 		const headerSpace = kintone.app.record.getHeaderMenuSpaceElement();
 		const cybozuAddress = [35.68704, 139.7861];
 		const name = record.Place.value;
 
-		headerSpace.style.height = '300px';
-		headerSpace.setAttribute('id', 'nihonbashi');
+		const mapSpace = document.createElement('div');
+		mapSpace.style.height = '300px';
+		mapSpace.setAttribute('id', 'nihonbashi');
+		headerSpace.appendChild(mapSpace);
 
-		mymap = L.map('nihonbashi').setView(cybozuAddress, 16);
+		const mymap = L.map('nihonbashi').setView(cybozuAddress, 16);
```

Comment:
You should rename `mymap` to a more meaningful name such as `cybozuMap`. (Sorry I should have mentioned this in the previous comment🙇‍♂️)